docs/flink-connector/flink-catalog-jdbc.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Flink connector jdbc catalog"
+slug: /flink-connector/flink-catalog-jdbc
+keyword: flink connector jdbc catalog
+license: "This software is licensed under the Apache License version 2."
+---
+
+This document provides a comprehensive guide on configuring and using Apache Gravitino Flink connector to access the Jdbc catalog managed by the Gravitino server.
+
+## Capabilities
+
+### Supported Jdbc Types
+
+* Mysql
+
+## Getting Started
+
+### Prerequisites
+
+Place the following JAR files in the lib directory of your Flink installation:
+
+- `flink-connector-jdbc-${flinkJdbcConnectorVersion}.jar`
+- `gravitino-flink-connector-runtime-1.18_2.12-${gravitino-version}.jar`
+- jdbc driver
+
+### SQL Example
+
+```sql
```

Comment:
Do we have some guide-style text for this page?
The whole subsection here is about showing that "this works". That is not what a guide is supposed to provide.

Reply:
The "Overview" already contains a "How to Use" section that explains how to use the Gravitino catalog in Flink. This section supplements the SQL syntax and options needed when using JDBC.